src/UI/app/views/DeckImport/DeckImport.html

```diff
@@ -1,10 +1,10 @@
 <div layout="column">
-  <deck-maker-header subtitle="'Import data'"></deck-maker-header>
+  <deck-maker-header subtitle="'Import Data'"></deck-maker-header>
   <ul layout="row" layout-align="center center" layout-padding>
     <md-card layout-padding flex="50">
       <md-card-content layout="column">
         <form ng-submit="submitForm()">
-          <md-card ng-repeat="error in errors" class="error">
+          <md-card ng-repeat="error in parsingErrors" class="error">
```

Comment:
this is a mistake, `parsingErrors` is a field on `Deck` but we're still populating our response with those parsingErrors saved to the `errors` field of the response.